wiki/Tournaments/AESS/1/en.md

```diff
@@ -0,0 +1,251 @@
+| 33-48 | ::{ flag=US }:: [Eribriel](https://osu.ppy.sh/users/12386362), ::{ flag=US }:: [eno](https://osu.ppy.sh/users/6658367), ::{ flag=KG }:: [Razak_](https://osu.ppy.sh/users/13195585), ::{ flag=US }:: [iamanewb](https://osu.ppy.sh/users/10350809), ::{ flag=US }:: [JigglingJerome](https://osu.ppy.sh/users/17091856), ::{ flag=BY }:: [Gadfour](https://osu.ppy.sh/users/9746556), ::{ flag=US }:: [minwork](https://osu.ppy.sh/users/14594272), ::{ flag=ID }:: [Contra](https://osu.ppy.sh/users/12011863), ::{ flag=UK }:: [Reshamen](https://osu.ppy.sh/users/17706086), ::{ flag=BE }:: [Ramfie](https://osu.ppy.sh/users/17201929), ::{ flag=MY }:: [Seushe](https://osu.ppy.sh/users/11234664), ::{ flag=NZ }:: [kit-](https://osu.ppy.sh/users/10981171), ::{ flag=US }:: [Remias](https://osu.ppy.sh/users/11782233), ::{ flag=CA }:: [atokymat](https://osu.ppy.sh/users/11465676), ::{ flag=NO }:: [Oribkens](https://osu.ppy.sh/users/12375176), ::{ flag=CA }:: [Bert](https://osu.ppy.sh/users/7330614) |
+| 49-64 | ::{ flag=UK }:: [MC AKIO](https://osu.ppy.sh/users/9008727), ::{ flag=US }:: [Chester Ming](https://osu.ppy.sh/users/4563687), ::{ flag=US }:: [Promooooooo](https://osu.ppy.sh/users/16614190), ::{ flag=US }:: [Azeylum](https://osu.ppy.sh/users/7873393), ::{ flag=HK }:: [kenny03316](https://osu.ppy.sh/users/2955661), ::{ flag=ID }:: [fnayR](https://osu.ppy.sh/users/2800253), ::{ flag=BR }:: [-LF-](https://osu.ppy.sh/users/11461810), ::{ flag=MA }:: [clorox\_1g](https://osu.ppy.sh/users/9702728), ::{ flag=TR }:: [Srpp](https://osu.ppy.sh/users/10087650), ::{ flag=US }:: [Tilenal](https://osu.ppy.sh/users/12086452), ::{ flag=CA }:: [Potato9756](https://osu.ppy.sh/users/13205755), ::{ flag=RU }:: [Liyachka](https://osu.ppy.sh/users/12777902), ::{ flag=MY }:: [MinaMyoui](https://osu.ppy.sh/users/6996926), ::{ flag=CA }:: [wolfflow11](https://osu.ppy.sh/users/14491879), ::{ flag=UK }:: [elix](https://osu.ppy.sh/users/10178296), ::{ flag=NZ }:: [mityaguy32](https://osu.ppy.sh/users/16536519) |
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | Player |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ::{ flag=CA }:: [FlatPaper](https://osu.ppy.sh/users/11255340) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ::{ flag=CA }:: [FaintGun](https://osu.ppy.sh/users/10500541) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ::{ flag=PH }:: [NathanSS](https://osu.ppy.sh/users/8500628) |
+
+## Mappools
```

Comment:
no mappack links? considering the non-uploaded map

Reply:
no mappacks since they were removed by mediafire since they were too old and unused or something :P